susemanager-utils/susemanager-sls/salt/util/mgr_switch_to_venv_minion.sls

```diff
@@ -0,0 +1,100 @@
+{%- set susemanager_conf='/etc/salt/minion.d/susemanager.conf' %}
+{%- set venv_susemanager_conf='/etc/opt/venv-salt-minion/minion.d/susemanager.conf' %}
+{%- set managed_minion=salt['file.file_exists'](susemanager_conf) and
+                       not salt['file.replace'](susemanager_conf, '^master: .*', 'master: ' + pillar['mgr_server'],
+                                                dry_run=True, show_changes=False, ignore_if_missing=True) %}
+{%- set venv_managed_minion=salt['file.file_exists'](venv_susemanager_conf) and
+                            not salt['file.replace'](venv_susemanager_conf, '^master: .*', 'master: ' + pillar['mgr_server'],
+                                                     dry_run=True, show_changes=False, ignore_if_missing=True) %}
+{%- if managed_minion or venv_managed_minion %}
+{%- set pkgs_installed = salt['pkg.info_installed']() %}
+{%- set venv_minion_installed = pkgs_installed.get('venv-salt-minion', False) and True %}
+{%- set venv_minion_available = venv_minion_installed or salt['pkg.latest_version']('venv-salt-minion') or False %}
+{%- if venv_minion_available %}
+venv-salt-minion-pkg:
```

Comment:
In theory we have the rule that all ids of states we write should start with `mgr_` . We do not follow this rule in all cases, but when we remember this, we should try to do it :-)

Reply:
Fixed